docs/src/Groups/permgroup.md

```diff
@@ -65,6 +67,30 @@ false
 ```
 In the example above, `x` and `y` are equal because both act on a set of cardinality `5`, while `x` and `z` are different because `x` belongs to `Sym(5)` and `z` belongs to `Sym(3)`.
 
+One can also create permutation group elements with the following syntax
+```@repl oscar
```

Comment:
I would recommend to use a `jldoctest` instead, whenever possible: you get a free test this way, and when reading the .md file it is immediately clear what the user will see in the rendered version of the manual.

Case in point, I've seen examples of  `@repl` being used where the rendered version said things like "syntax error"... because there indeed *was* a syntax error, which is easy to miss in the `.md` file

Reply:
All set now